src/components/plugin/glossary-popup-header.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
```

Comment:
Need to fix 4 space indents in this file - the standard is 2 for all CC projects.